src/ducks/filtrering.ts

```diff
@@ -124,6 +128,30 @@ export default function filtreringReducer(state: FiltervalgModell = initialState
             return {...action.data};
         case VELG_MINE_FILTER:
             return {...action.data.filterValg};
+        case FARGEKATEGORIER_HOVEDFILTER_KLIKK: {
+            const hovedfilterAlleredeValgt = state.ferdigfilterListe.includes(MINE_FARGEKATEGORIER);
+
+            const nyFerdigfilterListe = hovedfilterAlleredeValgt
+                ? state.ferdigfilterListe.filter(f => f !== MINE_FARGEKATEGORIER)
+                : [...state.ferdigfilterListe, MINE_FARGEKATEGORIER];
+            const nyFargekategorier = hovedfilterAlleredeValgt ? [] : [...alleFargekategoriFilterAlternativer];
+
+            return {...state, ferdigfilterListe: nyFerdigfilterListe, fargekategorier: nyFargekategorier};
+        }
+        case FARGEKATEGORIER_UNDERFILTER_KLIKK: {
```

Comment:
what is underfilter?

Reply:
Red = "Hovedfilter"
Green = "Underfilter"

I am open for suggestions for a better naming convention 😄 

![Screenshot 2024-03-18 at 10 53 07](https://github.com/navikt/veilarbportefoljeflatefs/
assets/111113539/5a5b022b-4df8-4b85-ae79-c29a47680071)

Reply:
![Screenshot 2024-03-18 at 10 53 07](https://github.com/navikt/veilarbportefoljeflatefs/assets/111113539/f884d689-b365-4cb1-9835-a744018cdad0)